README.md

```diff
@@ -57,30 +51,31 @@ class App extends React.Component {
 
 // remove static markup and allow React
 // to render only actual components
-removeUniversalPortals();
+prepareClientPortals();
 
 ReactDOM.render(<App />, document.querySelector("#root"));
 ```
 
 ```js
 // SERVER
 
-const { appendUniversalPortals } = require("react-portal-universal/lib/server");
+const { ServerPortal } = require("react-portal-universal/server");
 
-const body     = ReactDOMServer.renderToString(<App />));
+const portals  = new ServerPortal();
+const element  = portals.collectPortals(<App />);
+const body     = ReactDOMServer.renderToString(element));
 const template = fs.readFileSync(path.resolve("build/index.html"), "utf8");
 const html     = template.replace("<div id=\"root\"></div>", `<div id="root">${body}</div>`);
-const markup   = appendUniversalPortals(html);
+const markup   = portals.appendUniversalPortals(html);
 
 res.status(200).send(markup);
 ```
 
 ## Configure
 
 It is important to make sure that React application code is using the same instance of the library
-as code responsible for handling rendering on the server. In other words, there must be only one
-instance of the portals variable in the process. The problem occurs when you import
-`appendUniversalPortals` from `node_modules` on the server but use a bundle with its own instance to
+as code responsible for handling rendering on the server. The problem occurs when you import
+`ServerPortal` from `node_modules` on the server but use a bundle with its own instance to
```

Comment:
Do you find the sentence `In other words, there must be only one instance of the portals variable in the process." misleading or incorrect so you decided to remove it?

Reply:
I've added the sentence back in 👍